packages/light-apps/src/routing/identity.tsx

```diff
@@ -0,0 +1,20 @@
+// Copyright 2017-2018 @polkadot/light-apps authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { Route } from '../types';
+
+import { Identity } from '@polkadot/identity-app';
+
+export default (
+  {
+    Component: Identity,
```

Comment:
You are going here the centralized routing way (https://reacttraining.com/react-router/web/example/route-config), correct?

It's not recommended by RR4, but I'm not against it neither, just want to make sure.

Reply:
yes, I like how it allows App to be as minimal as possible. I am open to going about it differently if it is problematic however

Reply:
No it's fine, it's done the same on apps, so let's go this way.